index.html

```diff
@@ -16,13 +16,16 @@
             <div class="logo">
              <a href="index.html"><h2 class="hero">Mloman</h2></a>
             </div>
-            <ion-icon class="ion-icon hambuger" name="menu-outline" id="menu"></ion-icon>
+            <div class="hambuger-menu">
+                <i class="fa-solid fa-bars" id="menu"></i>
+                <!-- <i class="fa-solid fa-xmark hambuger" id="close"></i> -->
+            </div>
```

Comment:
- [ ] [optional] I would suggest adding some left padding for mobile menu options, It will make sure the design matches with the Figma design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/167638528-ed865b4c-abe4-43e1-b441-4ad9b66823e1.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167638676-c6bb7682-13fa-47ea-95a9-4937483a351c.png)